src/backend/lib/redis.js

```diff
@@ -1,11 +1,13 @@
 require('./config');
 const Redis = require('ioredis');
 const MockRedis = require('ioredis-mock');
+const url = require('url');
 
 const { logger } = require('../utils/logger');
 
 // If you need to set the Redis URL, do it in REDIS_URL
 const redisUrl = process.env.REDIS_URL || 'redis://127.0.0.1:6379';
+const { port, host } = url.parse(redisUrl, true);
```

Comment:
I think `url.parse()` can throw, so we should probably wrap this in a try/catch, and log an appropriate error before we crash.